.gitignore

```diff
@@ -10,3 +10,4 @@ shinyapps/
 README.html
 .*.Rnb.cached
 tools/yarn-error.log
+vignettes/*.R
```

Comment:
Why do these files need to be ignored?

Reply:
I have no idea. Generating vignettes seems to create a .R and a .html file in the vignettes directory. I cargo-culted this from dplyr (https://github.com/tidyverse/dplyr/blob/master/.gitignore#L12) and have just been manually ignoring/deleting the generated HTML file... 🙈

If you know better, I'm happy to revise.